ludwig/decoders/generic_decoders.py

```diff
@@ -146,7 +148,7 @@ def input_shape(self):
         return self.dense.input_shape
 
     def forward(self, inputs, **kwargs):
-        values = self.activation(self.dense(inputs))
+        values = self.activation(self.dense(inputs)) * self.multiplier
```

Comment:
I actually forgot we already had the activation! that was probably for doing softmax / sigmoid for vector features :)